devel/python/CMakeLists.txt

```diff
@@ -7,22 +7,23 @@ if (NOT DEFINED PYTHON_EXECUTABLE)
    return()
 endif()
 
+python_module(cwrap REQUIRED )
 
-python_module(numpy 1.7.1)
+python_module(numpy REQUIRED 1.7.1)
```

Comment:
> The user will get a warning if numpy is not installed, but everything (incl. Python) will be built.

Well - then you build a Python distribution which is not functional - I would prefer if Python was just skipped if the required libraries (i.e. `numpy` in this case) was not present.